src/components/Auth/ForgotPasswordForm.scss

```
@@ -2,6 +2,14 @@
   max-width: 400px;
   margin: 0 auto;
   padding: 24px;
+  // Matches .sign-in-form and .sign-up-form. AuthLayout used to paint an
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)